tests/test_data_processor.py

```diff
@@ -369,12 +478,64 @@ def test_max_altitude(self, data_processor):
         ],
     )
     def test_calculate_rotations(self, data_packets, expected_value):
```

Comment:
I want some more cases on this test. Another case should involve e.g. different axis of gravity vector, different gravity sign, no gyro data, etc.

Reply:
I'm gonna wait on this one for will